.github/workflows/dismiss-stale-approvals.yml

```diff
@@ -0,0 +1,25 @@
+name: Dismiss stale pull request approvals
+
+on:
+  pull_request:
+    types: [
+        opened,
+        synchronize,
+        reopened,
+      ]
+
+
+permissions:
+  actions: read
+  contents: read
+  pull-requests: write
+
+jobs:
+  dismiss_stale_approvals:
+    name: Dismiss stale pull request approvals
+    runs-on: ubuntu-latest
+    steps:
+      - name: Dismiss stale pull request approvals
+        uses: withgraphite/dismiss-stale-approvals@main
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Github is complaining about the missing newline at the end. 
![image](https://github.com/user-attachments/assets/3a759749-1eea-46c7-87e0-67572882e138)